src/main/scala/scalismo/io/MeshIO.scala

```diff
@@ -250,6 +250,16 @@ object MeshIO {
   private def readSTL(file: File, correctMesh: Boolean = false): Try[TriangleMesh[_3D]] = {
     val stlReader = new vtkSTLReader()
     stlReader.SetFileName(file.getAbsolutePath)
+
+    stlReader.MergingOn()
```

Comment:
If we choose MergingOff, then no points are merged, even not the identical ones which are only there because the same point is defined in two different triangles.